### PR TITLE
Added pyramid_openapi3

### PR DESCRIPTION
**Pull request by mandarvaze**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -238,13 +238,15 @@ based on [awesome-python](https://github.com/vinta/awesome-python/).
 * [nefertari](https://github.com/ramses-tech/nefertari) -  Nefertari is a REST
   API framework sitting on top of Pyramid and ElasticSearch.
 * [pyramid_swagger](https://github.com/striglia/pyramid_swagger) - Convenient
-  tools for using Swagger to define and validate your interfaces in a Pyramid webapp.
+  tools for using Swagger to define and validate your interfaces in a Pyramid webapp. (SWagger 2.0 document)
+* [pyramid-openapi3](https://github.com/niteoweb/pyramid_openapi3) - Validate Pyramid views against an OpenAPI 3.0 document. Similar to pyramid_swagger but for openapi3
```
Comment (stevepiercy) [inline README.md]: FWIW, we are in the process of migrating this repo under the Pylons Project.

Also "openapi3" should be "OpenAPI 3.0".

Add a period to the end of the sentence.

File: README.md [hunk 2]
```diff
@@ -238,13 +238,15 @@ based on [awesome-python](https://github.com/vinta/awesome-python/).
 * [nefertari](https://github.com/ramses-tech/nefertari) -  Nefertari is a REST
   API framework sitting on top of Pyramid and ElasticSearch.
 * [pyramid_swagger](https://github.com/striglia/pyramid_swagger) - Convenient
-  tools for using Swagger to define and validate your interfaces in a Pyramid webapp.
+  tools for using Swagger to define and validate your interfaces in a Pyramid webapp. (SWagger 2.0 document)
```
Comment (stevepiercy) [inline README.md]: lower case "w" in Swagger.

File: README.md (hunk 1 above)
Comment (mandarvaze) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (mandarvaze) [reply README.md]: Done.

**Comment by uralbash** [comment]
Thanks!

**Pull request merged**